docs/plone-deployment/new-project.md

```diff
@@ -142,3 +141,18 @@ When the process completes successfully, it will exit with a message similar to
 ```console
 ✨  Done in 98.97s.
 ```
+
+Remember, having the repository on GitHub is currently a requirement to build the Docker images automatically; this might be a good moment to do so in case you had not yet.
+
+The same could be done with other options like GitLab, but we do not provide a working example for it at the moment.
```

Comment:
We do not want to imply that the authors will write working examples in the trainings. If you have a working example that you would like to share, then please include it in this PR or another.

```suggestion
```